### ISSUE TITLE
Зависимость от ночной сборки

### ISSUE THREAD

**Issue by 240596448** [oscript-library/opm#153]
Обновляю все пакеты и получаю такую ошибку
```
ИНФОРМАЦИЯ - Скачиваю файл: edt-export-bugs.ospx
ИНФОРМАЦИЯ - Устанавливаю пакет edt-export-bugs
ПРЕДУПРЕЖДЕНИЕ - Обрабатываю возникшую ошибку...
{Модуль C:\OneScript\lib\opm\src\core\Классы\МенеджерУстановкиПакетов.os / Ошибка в строке: 115 / {Модуль C:\OneScript\lib\opm\src\core\Классы\УстановкаПакета.os / Ошибка в строке: 112 / Ошибка установки пакета <edt-export-bugs>: Обнаружена устаревшая версия движка OneScript.
Требуемая версия: 1.1
Текущая версия: 1.0.21.1
Обновите OneScript перед установкой пакета}}
```
Полагаю, что налицо противоречие между версиями.... ? Не должен установщик тянуть несовместимые версии. Или же, если сообщение задуманное, то его текст должен быть несколько иным.
Еще мне казалось, что где-то должен был быть ключ выбора канала обновления (stable/dev) или мне это почудилось?

**Comment by Stepa86** [comment]
Да, этот пакет работает только на ночнике и не работает на стабильной релизе. Достаточно не ставить все пакеты оптом, чтобы не получать такой ошибки

**Comment by 240596448** [comment]
Ошибку вижу в конце "портянки". Вопрос: на ошибке установка прерывается или просто информационные сообщения выводятся в конце (что сомнительно) ?
Если прерывается, то значит проблемный пакет делает невозможной установку `--ALL`.
Логика подсказывает, что установка "оптом" должна как-то отбирать только подходящие пакеты.

**Comment by ander77** [comment]
Поддержу. Мне тоже удобнее поставить все пакеты и по ним уже смотреть, что есть интересного и полезного, чем упираться "рогом" несколько дней в проблему, а потом обнаружить что готовое решение уже кем-то написано.
